puresnmp/x690/types.py

```diff
@@ -93,7 +104,8 @@ def pop_tlv(data):
     return value, remainder[length:]
 
 
-class Type(metaclass=Registry):
+@six.add_metaclass ( Registry )
```

Comment:
Can you run a PEP-8 checker on your code? I've seen these spurious spaces around identifiers and parens on various places which should be removed to have a more consistent code-style

Reply:
There are things about PEP-8 that really stick in my craw, so it's not something I following closely in my personal projects. Unfortunately it means I'm not very good at it when submitting patches to other projects. I will look into how to run a PEP-8 check and do that.

Reply:
I get that. It is difficult when a tool imposes a code-style onto you that you don't like. I have my issues with certain things in PEP8 as well. Then again, PEP8 itself statees that it is only a guideline and that you can break it if you have a good reason. After all, it's about readability. And sometimes PEP8 *does* make the code less readable.

Having said that, the issue I have with the spaces around the functions is primarily that it is inconsistent with the rest of the code in this project 😉 

One advantage here of PEP8 is that there are many checkers available which you can easily run.

Reply:
yes and I fixed up most of the spaces that I had introduced, but I just apparently missed some. As I said I had not intended to break the coding style of the project in any way, I just wasn't careful enough. I'm working on it now.